src/mappingForProton.ts

```diff
@@ -31,6 +31,7 @@ import { loadOrCreateApprovedOperator } from './helpers/loadOrCreateApprovedOper
 
 import { ZERO, ADDRESS_ZERO, NEG_ONE, getStringValue, getBigIntValue } from './helpers/common';
 import { loadOrCreateGenericRoyaltiesClaimedByAccount } from './helpers/loadOrCreateRoyaltiesClaimedByAccount';
+import { updateNftAnalytics } from './helpers/updateNftAnalytics';
```

Comment:
this should be a "loadOrCreateNftAnalytics" first, and then set the updated values in the mapping file

Reply:
Fixed